platform/conf/conf.go

```diff
@@ -232,7 +232,7 @@ func (u *UserData) Render(ctPlatform string) (*Conf, error) {
 			plog.Warningf("rendering Container Linux config: %s", report)
 		}
 
-		c.ignitionV21 = &ignc
```

Comment:
I'd break this into a seperate commit, otherwise LGTM

Reply:
Yeah, that'd be my preference, but I didn't want to break the build for one commit.

Reply:
At least call it out in the commit message?

Reply:
Done.